app/src/main/res/values/strings.xml

```diff
@@ -929,4 +929,5 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="dialog_tutorial_download">With auto-sync off, quests are not downloaded automatically. To do this manually, use the \"%s\" button in the menu at the location you want to download.</string>
 
     <string name="quest_pedestrian_crossing_island">"Does this pedestrian crossing have an island?"</string>
+    <string name="quest_tactile_paving_kerb">Is there a tactile paving on this kerb?</string>
```

Comment:
Minor tweak
```suggestion
    <string name="quest_tactile_paving_kerb">Is there tactile paving on this kerb?</string>
```

Is this supposed to be all the way along a kerb or just at a crossing, in which case isn't it covered by the existing quest?

Reply:
Just at the specific point of a specific crossing (in extreme cases one side of crossing may have tactile paving, second not).

Reply:
What about 

> Is there tactile paving on kerb here?

?